src/Makefile

```
@@ -516,6 +523,7 @@ test-cluster: $(SERVER_NAME) $(ENGINE_CLI_NAME)
 check: test
 
 lcov:
+	$(call ERROR_IF_LCOV_NOT_INSTALLED)
```

I think this approach adds a lot of complexity in the Makefile. We want to keep it simple and readable.

What's wrong with just printing the version as the first command here? If it fails, then the run does not start.

```suggestion
	@lconv --version
```

---

At first I thought we could be a little more precise with our hints, but maybe it's good to keep it simple here.

---

Yeah, in this case, I think it's not worth the extra complexity.

If we want to print a message, we can do something like

    @lconf --version || echo "lcov is not installed" && false